src/rql/index.ts

```diff
@@ -20,8 +26,26 @@ export function rqlParser(rql: string): RQLString {
  * @see https://developers.extrahorizon.io/guide/rql.html
  * @returns
  */
-export function rqlBuilder(rql?: RQLString | string): RQLBuilder {
-  let returnString = rql && rql.charAt(0) === '?' ? rql.substr(1) : rql || '';
+export const rqlBuilder: RqlBuilderFactory = (
+  input: RQLBuilderInput
+): RQLBuilder => {
+  let returnString = '';
+  let shouldDoubleEncode = rqlBuilder.doubleEncodeValues;
+
+  let rqlString: RQLBuilderString;
+  if (typeof input === 'object') {
```

Comment:
(minor) nitpicky, `null` is also considered an object. If `null` is supplied, `const { doubleEncode, rql } = input;` will fail, I belief 🤔